spec/spec_helper.rb

```diff
@@ -1,4 +1,20 @@
 require 'pathname'
+
+require 'simplecov'
+
+SimpleCov.start do
+  track_files 'lib/**/*.rb'
+  add_filter 'lib/reek/version.rb' # version.rb is loaded too early to test
+end
+
+if ENV['FAIL_ON_LOW_COVERAGE'] == '1'
+  SimpleCov.at_exit do
+    SimpleCov.result.format!
+    SimpleCov.minimum_coverage 98.71
```

Comment:
Seems having these values will lead to a lot of churn of this file.

One alternative approach I like is to specify 100% coverage and use `# :nocov:` markers for the lines to be ignored.